tests/migration/remove_dropped_modules.pm

```diff
@@ -0,0 +1,24 @@
+# SLE15 migration tests
+#
+# Copyright 2016-2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: SUSEConnect
+# Summary: Some modules may not available on the target product, remove these modules before migration
```

Comment:
Some modules don't exist on the target product any more, need remove these modules before migration. WDYT?

Reply:
ok for me.